backend/src/main/java/de/ffmjava/capstone/backend/horses/HorseController.java

```diff
@@ -19,4 +22,13 @@ public class HorseController {
     public List<Horse> getAllHorses() {
         return service.getAllHorses();
     }
+
+    @PostMapping
+    @ResponseStatus(code = HttpStatus.CREATED)
+    public ResponseEntity<Object> addNewHorse(@Valid @RequestBody Horse newHorse, Errors errors) {
+        ResponseEntity<Object> errorMessage = CustomApiErrorHandler.handlePossibleErrors(errors);
+        if (errorMessage != null) return errorMessage;
+        Horse createdHorse = service.addNewHorse(newHorse);
+        return new ResponseEntity<>(createdHorse, HttpStatus.CREATED);
+    }
```

Comment:
@ResponseStatus  annotation is redundant here, as you already return Response Entity with HTTP status.